app/models/local_authority.rb

```diff
@@ -0,0 +1,5 @@
+class LocalAuthority < ActiveRecord::Base
+  validates :gss, :snac, uniqueness: true
+  validates :gss, :name, :snac, :tier, presence: true
+  validates :homepage_url, format: { with: URI.regexp }, allow_blank: true
+end
```

Comment:
We should also validate that `tier` is one of `%w{district county unitary}`, for completeness.